### PR TITLE
[RFC] Introduce a PathBuilder helper

### PR DESCRIPTION
**Pull request by lchrusciel**
During #67 I found it really annoying to carry about `DIRECTORY_SEPARATOR`, so I decide to do something about it. But with this implementation, a php version bump to 5.6 is require, thats why I have  add a RFC flag. Please, share your thoughts, should it be added or not.

### CODE REVIEW

File: src/PathBuilder.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Lakion\ApiTestCase;
+
+/**
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+class PathBuilder
```
Comment (pamil) [inline src/PathBuilder.php]: abstract final class PathBuilder

File: composer.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
         }
     ],
     "require": {
-        "php": "^5.5.9|^7.0",
+        "php": "^5.6|^7.0",
```
Comment (pamil) [inline composer.json]: :tada:

File: src/PathBuilder.php (hunk 1 above)
Comment (lchrusciel) [inline src/PathBuilder.php]: Of course!

**Comment by pamil** [comment]
https://travis-ci.org/Sylius/Sylius/builds/146588761 [*]

**Pull request merged**